pulpcore/cli/container/context.py

```diff
@@ -75,6 +76,8 @@ class PulpContainerRepositoryContext(PulpRepositoryContext):
     UPDATE_ID = "repositories_container_container_partial_update"
     DELETE_ID = "repositories_container_container_delete"
     SYNC_ID = "repositories_container_container_sync"
+    TAG_ID: ClassVar[str] = "repositories_container_container_tag"
+    UNTAG_ID: ClassVar[str] = "repositories_container_container_untag"
     VERSION_CONTEXT = PulpContainerRepositoryVersionContext
     CAPABILITIES = {
         "sync": [PluginRequirement("container")],
```

Comment:
If adding/removing tags between the two repository types is supported from different versions of pulp_container then I can add capabilities here. Might be worth to do it anyway since I don't know which version tagging was added in.

Reply:
Support for tagging manifests within push repositories was added in version 2.3.0 (2021-02-08). According to the pulp-cli [docs](https://github.com/pulp/pulp_container/blob/2.3/requirements.txt#L1), we support `5 versions of pulpcore from 3.11-3.16 and 5 of Pulp's plugins: pulp_ansible, pulp_container`. Version 2.3.0 falls into the pulpcore requirement when looking at [requirements.txt](https://github.com/pulp/pulp_container/blob/b81d233f0ce950153cfea9165cd7b4c2aad3286f/requirements.txt#L1).